### PR TITLE
Add http2 monitoring documentation

### PR DESCRIPTION
**Pull request by amitslavin**
### What does this PR do? What is the motivation?
Adds documentation to enable USM to capture HTTP/2 and gRPC traffic.

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

### CODE REVIEW

File: content/en/universal_service_monitoring/setup.md [hunk 1]
```diff
@@ -849,6 +849,46 @@ agents:
 
 {{< /tabs >}}
 
+### HTTP/2 Monitoring
+
+<div class="alert alert-info">
+Universal Service Monitoring is available in <strong>beta</strong> to capture <a href="https://datatracker.ietf.org/doc/html/rfc7540">HTTP/2</a> traffic.
```
Comment (guyarb) [inline content/en/universal_service_monitoring/setup.md:855]: Not beta

**Review by guyarb — CHANGES_REQUESTED** [review]

**Comment by guyarb** [comment]
Also, you should add to the description - this feature allows USM to capture HTTP/2 and gRPC traffic

File: content/en/universal_service_monitoring/setup.md [hunk 2]
```diff
@@ -851,9 +851,7 @@ agents:
 
 ### HTTP/2 Monitoring
 
-<div class="alert alert-info">
-Universal Service Monitoring is available to capture <a href="https://datatracker.ietf.org/doc/html/rfc7540">HTTP/2</a> traffic.
-</div>
+Universal Service Monitoring is available to capture HTTP/2 traffic.
```
Comment (guyarb) [inline content/en/universal_service_monitoring/setup.md:854]:
```suggestion
Universal Service Monitoring is available to capture HTTP/2 and gRPC traffic.
```

File: content/en/universal_service_monitoring/setup.md [hunk 3]
```diff
@@ -849,6 +849,44 @@ agents:
 
 {{< /tabs >}}
 
+### HTTP/2 Monitoring
```
Comment (buraizu) [inline content/en/universal_service_monitoring/setup.md:852]:
```suggestion
### HTTP/2 monitoring
```

File: content/en/universal_service_monitoring/setup.md [hunk 4]
```diff
@@ -849,6 +849,44 @@ agents:
...
+
+Universal Service Monitoring is available to capture HTTP/2 and gRPC traffic.
```
Comment (buraizu) [inline content/en/universal_service_monitoring/setup.md:854]:
```suggestion
Universal Service Monitoring can capture HTTP/2 and gRPC traffic.
```

File: content/en/universal_service_monitoring/setup.md [hunk 5]
```diff
@@ -849,6 +849,44 @@ agents:
...
+```
+
+{{% /tab %}}
+{{% tab "Environment variable" %}}
+
+```conf
+DD_SERVICE_MONITORING_CONFIG_ENABLE_HTTP2_MONITORING=true
+```
+{{% /tab %}}
+
```
Comment (buraizu) [inline content/en/universal_service_monitoring/setup.md:875]:
```suggestion
```

**Review by buraizu — APPROVED** [review]
Approving with some minor edits suggested

**Pull request merged**